roboteam_ai/src/skills/SideAttacker.h

```diff
@@ -19,7 +19,7 @@ class SideAttacker : public Skill {
     Vector2 deltaPos;
     Vector2 targetPos;
 
-    int zone = -1;
+    int zone;
```

Comment:
looks like a testing artifact to me that you removed this, was there a good reason?

Reply:
Removed it, all this logic is moved to the OffensiveCoach